FhirLoader.Tests.E2E/UnitTest.cs

```diff
@@ -0,0 +1,83 @@
+using FhirLoader.Tool;
+
+namespace FhirLoader.Tests.E2E
+{
+    public class UnitTest
+    {
+        private const string fhirURL = "https://workpacesdkfhir-sjbuildfhir.fhir.azurehealthcareapis.com/";
+        private const int batchsize = 500;
+        private const int concurrency = 50;
+
+        /// <summary>
+        /// E2E Test Case for bundle Files.
+        /// By passing FolderPath,it will read all the files and processed to the Fhir URL. 
+        /// </summary>
+        [Fact]
+        public async void LocalFhir_bundle_Test()
+        {
+            CommandOptions commandOptions = new()
+            {
+                FhirUrl = fhirURL,
+                FolderPath = @"E:\Projects\fhir-loader\TestSamples\bundle",
```

Comment:
All paths need to be relative to the test so it works on other people's machines.

Reply:
Done.